examples/window-app/common/include/WindowApp.h

```diff
@@ -157,6 +157,7 @@ class WindowApp
 
     virtual ~WindowApp() = default;
     virtual CHIP_ERROR Init();
+    virtual CHIP_ERROR StartAppTask();
```

Comment:
Called StartApptask function from the main() which will create AppTaskMain task and the Event Queue.
The AppTaskMain will initializing the window application and run the event handler loop.Removed the redundant task exist before and now having only one task and the flow is similar for both WiFi and Thread. 
The design flow is similar to the Light and door-lock apps except the base application class reference.

Reply:
@jepenven-silabs   In main function calling startAppTask()  through window object called "app" hence we declare and define virtual startAppTask() in windowApp.h and windowApp.cpp file respectively

If removed it will get linking error

Reply:
The goal is to remove it completely since it breaks the abstraction layer. There shouldn't be any reference to any AppTask anywhere in this demo...